src/poliastro/twobody/orbit.py

```diff
@@ -8,7 +8,7 @@
 
 from poliastro.constants import J2000
 from poliastro.twobody.angles import nu_to_M
-from poliastro.twobody.propagation import propagate
+from poliastro.twobody.propagation import propagate, cowell
```

Comment:
Where are you using the `cowell` function? And on the other hand, do you plan to add it for testing?

Reply:
Yeah! I plan to do so. Currently, not using it